### DISCUSSION TITLE
Install requirements.txt secrets

### DISCUSSION THREAD

**Discussion by paxzeno** [skills/secure-code-game#48] [Feedback]
Hi all,
When I try to install the requirements I'm getting this error:
_

> ImportError: Installing this module requires OpenSSL python bindings

_

The issue is when installing the secrets module, but it's a bit odd, since to my knowledge the module is already packaged with python.
I installed OpenSSL with brew, since I'm on a Mac.

This doesn't seem to be a blocker, just wanted to give this feedback

**Comment by jkcso** [comment]
Thanks a lot! I'm looking into it and can't reproduce, were you on a codespace?
